tests/integration/test_plotter.py

```diff
@@ -657,3 +657,27 @@ def test_name_filter(modeler: Modeler, verify_image_cache):
         filter="Cyl",
         screenshot=Path(IMAGE_RESULTS_DIR, "test_name_filter.png"),
     )
+
+
+def test_plot_design_point(modeler: Modeler, verify_image_cache):
```

Comment:
Ideally - add a docstring for the test. Also add it to the one for name filters. Just realized it was missing